src/Sylius/Bundle/TaxonomyBundle/Form/EventListener/BuildTaxonomyFormListener.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\TaxonomyBundle\Form\EventListener;
+
+use Sylius\Component\Taxonomy\Model\TaxonInterface;
+use Sylius\Component\Taxonomy\Model\TaxonomyInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\Form\FormEvent;
+use Symfony\Component\Form\FormEvents;
+use Symfony\Component\Form\FormFactoryInterface;
+
+/**
+ * Adds the taxonomy name to the root taxon
+ *
+ * @author Andy Clyde <me@andyclyde.com>
+ */
+class BuildTaxonomyFormListener implements EventSubscriberInterface
```

Comment:
Well, yes, it does implement the EventSubscriberInterface. But then it's in a directory called \Sylius\Bundle\TaxonomyBundle\Form\EventListener and the other class in that dir is \Sylius\Bundle\TaxonomyBundle\Form\EventListener\BuildTaxonFormListener so I was trying to keep it consistent! Should we rename the whole dir and both classes? Or just leave it as is?